### ISSUE TITLE
problems with date

### ISSUE THREAD

**Issue by kschaerer** [pods-framework/pods#5407]
## Issue Overview

the date component changes the date I assign, for example, instead of accepting the 2019/07/06 date, it changes it to 2019/07/07, once I save the pod, and then I can not change it.

I tried to change by placing a previous date, but it does not allow me to edit.

observation: the date settings in wordpress are correct, all the other functions of wordpress or other plugins are working well, only the date of pods is altered

## Expected Behavior

It should save the date as I write and not change it

It should also allow myself to edit

 

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## WordPress Environment

<details>
```
Copy and paste your System Details from **Pods Admin > Settings > Debug Information** in WordPress admin here.
```
</details>

## Pods Package Export (helpful!)

```
Copy and Paste the JSON Export from **Pods Admin, Migrate: Packages, Export** in WordPress admin here
```

**Comment by JoryHogeveen** [comment]
@kschaerer 
Could you please check the latest update and let us know if #5409 fixed this?
If not, please test this PR: #5418

**Comment by JoryHogeveen** [comment]
Closing this due to no reply. Feel free to reopen if the problem is not fixed after we update to 2.7.15.
